app/api/v1/endpoints/memory/endpoints.py

```
@@ -178,6 +186,55 @@ async def get_memory_asset(
+    description="Returns a short-lived signed URL for a stored memory asset.",
+    responses={
+        200: {"description": "Signed URL generated"},
+        401: {"description": "Authentication required"},
+        404: {"description": "Asset not found"},
+    },
+)
+async def get_memory_asset_signed(
+    bucket: str,
+    object_path: str,
+    _current_user: Annotated[TokenPayload, Depends(get_current_user)],
+    supabase: SupabaseClient = Depends(get_supabase_client),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)